src/Comments.tsx

```diff
@@ -0,0 +1,33 @@
+import * as types from './types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faFaceSmile } from '@fortawesome/free-solid-svg-icons';
+import {comments as initialComments}  from './sampleData'
+import { useEffect} from 'react';
+
+
+export const Comments: React.FC<types.commentsProps> = ({ comments = initialComments, setComments }) => {
```

Comment:
It may make sense to create a `src/components` folder to put components like this